solidity/contracts/liquidity-protection/LiquidityProtectionSettings.sol

```diff
@@ -276,12 +275,7 @@ contract LiquidityProtectionSettings is ILiquidityProtectionSettings, AccessCont
      *
      * @param poolAnchor    pool anchor
      */
-    function addPoolToWhitelist(IConverterAnchor poolAnchor)
-        external
-        onlyOwner
-        validAddress(address(poolAnchor))
-        notThis(address(poolAnchor))
-    {
+    function addPoolToWhitelist(IConverterAnchor poolAnchor) external onlyOwner validAddress(address(poolAnchor)) {
```

Comment:
shouldn't be `validExternalAddress`?

Reply:
I think that the idea is to get rid of the 'notThis' check almost everywhere

Reply:
Yeah, we don't really need to do `validExternalAddress` here, because an incorrect setting can be reverted via the `remove` method

Reply:
isn't the idea to prevent the setting from getting added in the first place?

Reply:
:+1: